.github/workflows/run-tests.yml

```diff
@@ -390,12 +393,11 @@ jobs:
       TEST_SHARD_INDEX: ${{ matrix.shard_index }}
       PERSISTENCE_TYPE: ${{ matrix.persistence_type }}
       PERSISTENCE_DRIVER: ${{ matrix.persistence_driver }}
-      SINGLE_TEST_ARGS: ${{ needs.set-up-single-test.outputs.single_test_args }}
       TEST_TIMEOUT: ${{ needs.set-up-single-test.outputs.test_timeout }}
       BUILDKITE_MESSAGE: '{"job": "functional-test", "db": "${{ matrix.persistence_driver }}"}'
     steps:
       - name: Print functional test
-        run: echo "${{ needs.set-up-single-test.outputs.dbs }}" && echo "$SINGLE_TEST_ARGS"
+        run: echo "${{ needs.set-up-single-test.outputs.dbs }}" && echo "${{ needs.set-up-single-test.outputs.single_test_args }}"
```

Comment:
do we really need this? why not just have the thing that runs functional tests log the exact command it's using